tasks/bitsight/spec/bitsight_spec.rb

```diff
@@ -141,11 +141,11 @@
       end
 
       it "should translate api results into vali kdi" do
-        guid = get_my_company(@key)
+        # guid = get_my_company(@key)
 
-        results = get_bitsight_findings_and_create_kdi(@key, guid, 100)
+        # results = get_bitsight_findings_and_create_kdi(@key, guid, 100)
```

Comment:
I think these 2 lines are needed but the second one does not need to return back to a variable.